src/test/java/com/hei/wallet/wallety/controller/TransactionControllerIT.java

```diff
@@ -0,0 +1,60 @@
+package com.hei.wallet.wallety.controller;
+
+import com.hei.wallet.wallety.endpoint.rest.controller.TransactionController;
+import com.hei.wallet.wallety.model.Transaction;
+import com.hei.wallet.wallety.repository.TransactionRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.math.BigDecimal;
+import java.sql.SQLException;
+import java.time.Instant;
+import java.util.List;
+
+@SpringBootTest
```

Comment:
TransactionTest should be discuss before doing thus 'cause their are a lot of thing to test (same for transfer)